gluten-core/src/main/scala/org/apache/spark/sql/hive/HiveTableScanExecTransformer.scala

```diff
@@ -79,6 +80,10 @@ class HiveTableScanExecTransformer(
     Seq.empty
   }
 
+  override def getRootPathsInternal: Seq[Path] = {
+    relation.tableMeta.storage.locationUri.map(new Path(_)).toSeq
```

Comment:
Thanks for your work!
Could you clarify the overhead required for validating root paths? Does each file have a root path?

Reply:
I think the overhead of rootPath is smaller, at least it doesn't need to be listed. For nodes without rootPath we will pass conversion.